docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
```

Comment:
It actually doesn't, it's an issue I described as part of https://krausefx.com/blog/scaling-open-source-communities (long post, no need to read it all)

Reply:
I'm not sure i understand. If the CI bot comments on the PR does the user not get a notification?